script.js

```diff
@@ -0,0 +1,201 @@
+// All the DOM selectors stored as short variables
+const chat = document.getElementById("chat");
+const inputWrapper = document.getElementById("input-wrapper");
+const input = document.getElementById("input");
+const sendBtn = document.getElementById("send");
+
+let questionNumber = 1;
+
+const botReply = (msg) => {
```

Comment:
Nice that you went the extra mile and created a bot- and user-reply function. This makes your code more readable.